code/controllers/subsystems/records.dm

```diff
@@ -0,0 +1,137 @@
+/var/datum/controller/subsystem/records/SSrecords
+
+/datum/controller/subsystem/records
+    name = "Records"
+    flags = SS_NO_FIRE
+
+    var/list/records
+
+/datum/controller/subsystem/records/New()
+    LAZYINITLIST(records)
```

Comment:
Might as well just have `records = list()`, no real benefit to doing it like this on a global singleton.